tests/rbd_mirror/test_image_removal_from_secondary.py

```diff
@@ -0,0 +1,64 @@
+from tests.rbd_mirror.rbd_mirror_utils import rbd_mirror_config
+from utility.log import Log
+
+log = Log(__name__)
+
+
+def test_image_removal_from_secondary(rbd_mirror, pool_type, **kw):
+    try:
+        mirror1 = rbd_mirror.get("mirror1")
+        mirror2 = rbd_mirror.get("mirror2")
+        config = kw.get("config")
+        pool = config[pool_type]["pool"]
+        image = config[pool_type]["image"]
+        imagespec = pool + "/" + image
+
+        # verify for journal mirroring enable
+        if mirror1.get_mirror_mode(imagespec) == "journal":
+            log.info("journal mirroring is enabled")
+
+        # Disable journal mirroring on image at primary
+        mirror1.disable_mirroring("image", imagespec)
+
+        # verify removal from secondary
+        if mirror2.image_exists(imagespec):
+            return 0
```

Comment:
Return value is confusing. When image exists it should return 1 I suppose

Reply:
Also please add a log.error here describing the error if image exists after disabling

Reply:
> Return value is confusing. When image exists it should return 1 I suppose
if image exist the function will return 0.